lib/EventTimeoutProcessor.ts

```diff
@@ -11,6 +12,10 @@ export class EventScheduler implements IProcessor {
     }
 
     public async processor(jobContext: IJobContext, read: IRead, modify: IModify, http: IHttp, persis: IPersistence): Promise<void> {
+
+        const response: IDialogflowMessage = await Dialogflow.sendRequest(http, read, modify, jobContext.rid, jobContext.event, DialogflowRequestType.EVENT);
+
+        await createDialogflowMessage(jobContext.app, jobContext.rid, read, modify, response);
```

Comment:
The first parameter in the fn is IApp, in the jobContext it is not available. So it can break if we try to use (app: IApp) in the fn. 

I tried using logger in createMessage fn

<img width="1330" alt="Screenshot 2021-08-19 at 8 07 56 PM" src="https://user-images.githubusercontent.com/18264684/130088221-cc58f755-f804-46c6-8d0d-37223ece36a3.png">

<img width="1792" alt="Screenshot 2021-08-19 at 8 04 22 PM" src="https://user-images.githubusercontent.com/18264684/130088255-e5a53ca8-b89b-4c42-8c3d-7cd194d80da7.png">

And it didn't send the message after timeout

Reply:
@AlexanderKanakis has this been resolved? thanks!

Reply:
@chadgoss I need some time to figure out why passing app inside the jobcontext breaks the app.

Reply:
👍